ui/page/send_confirm_modal.go

```diff
@@ -1,34 +1,60 @@
-package ui
+package page
```

Comment:
Is there a reason this modal isn't in the modal package?

Reply:
Yes, it’s supposed to be grouped with send page(and other send page files) in a folder since it’s not a shared modal.